Packs/DelineaSS/Integrations/DelineaSS/DelineaSS.py

```diff
@@ -47,8 +48,8 @@ def getUsernameById(self, secret_id: str) -> str:
         url_suffix = "/api/v1/secrets/" + str(secret_id) + "/fields/username"
         return self._http_request("GET", url_suffix)
 
-    def getSecret(self, secret_id: str) -> str:
-        url_suffix = "/api/v1/secrets/" + str(secret_id)
+    def getSecret(self, secret_id: str, autocomment: str = '') -> str:
+        url_suffix = "/api/v1/secrets/" + str(secret_id) + "?autoComment=" + str(autocomment)

```

Comment:
You can pass this via the `params` argument for `_http_request` - that way `autocomment` URL argument will not be passed at all when it's not set.